composer.json

```diff
@@ -13,7 +13,9 @@
 	],
 	"require": {
 		"php": ">=7.2",
-		"composer/installers": "^2"
+		"composer/installers": "^2",
+		"ext-dom": "*",
+		"ext-libxml": "*"
```

Comment:
Are these required PHP extensions? What happens if they're not installed in the environment? Is this a possibility at all?